Strategy11/formidable-forms
# Implement Deactivate and Uninstall Buttons for Add-Ons on Formidable Add-Ons Page

This PR addresses the feature request in [Issue #4349](https://github.com/Strategy11/formidable-pro/issues/4349), which involves enhancing the user interface of the Formidable Add-Ons page (`/wp-admin/admin.php?page=formidable-addons`). The update adds "Deactivate" and "Uninstall" buttons for add-ons, providing clearer options for managing their states.

## Related Issue:
[Add Deactivate and Uninstall buttons to the Formidable Add-Ons page #4349](https://github.com/Strategy11/formidable-pro/issues/4349)

## Testing Instructions:
1. Navigate to `WP Admin > Formidable > Add-Ons`.
2. For each add-on, verify the presence of the appropriate button(s) based on its current state:
   - "Deactivate" when the add-on is active.
   - "Uninstall" when the add-on is deactivated.
3. Test the functionality of the new buttons to ensure they correctly deactivate and uninstall the add-ons.

## Formidable PRO Plugin PR:
https://github.com/Strategy11/formidable-pro/pull/4676

## Output:
![2023-12-27 20 38 07](https://github.com/Strategy11/formidable-forms/assets/69119241/f0852da4-33c6-4573-9fa3-efe7619f9b46)
